lib/response_massager.js

```diff
@@ -43,6 +43,7 @@ class ResponseMassager {
           hit._source.electronicResources = hit.inner_hits.electronicResources.hits.hits
             .map((resource) => resource._source.electronicLocator)
             .reduce((acc, el) => acc.concat(el), [])
+            .filter((resource) => !resource.url.includes('aeon'))
```

Comment:
This is too permissive. We don't want to remove links that happen to include phrase. Use [this](https://github.com/NYPL/discovery-front-end/blob/60d18724234177133ed5ef35d821e9b423f8057f/src/server/utils/isAeonUrl.js#L11-L19) set of `startsWith` checks